mlflow/utils/_capture_modules.py

```diff
@@ -120,7 +121,7 @@ def _load_pyfunc_patch(*args, **kwargs):
             with cap_cm:
                 model = original(*args, **kwargs)
                 if input_example is not None:
-                    model.predict(input_example)
+                    model.predict(input_example, params=params)
```

Comment:
I think we need a test for guarding this line,
e.g. test it captures dependencies that are used by model inference with specific params

Reply:
I don't think params will introduce extra dependencies? 🤔

Reply:
I don't think so, we allow custom models and some custom model params might cause extra package imported